.changeset/common-goats-exist.md

```diff
@@ -0,0 +1,25 @@
+---
+'@backstage/plugin-notifications-backend': patch
+---
+
+Notifications are now automatically deleted after 1 year by default.
+
+There is a new scheduled task that runs every 24 hours to delete notifications older than 1 year.
+This can be configured by setting the `notifications.expiration.period` in the `app-config.yaml` file.
+
+```yaml
+notifications:
+  expiration:
+    period: 365d # Default is 365 days
+```
+
+If the period is set to `0`, the expiration feature will be disabled, and notifications will not be automatically deleted.
+
+The scheduled task can also be scheduled to run at a different interval by setting the
+`notifications.expiration.schedule` in the `app-config.yaml` file.
+
+```yaml
+notifications:
+  expiration:
+    schedule: '0 0 * * *' # Default is every 24 hours
+```
```

Comment:
Could you add this to `docs/notifications` instead? There's a lot of useful info here that will get lost in the changelog.